services/github/src/handlers.js

```diff
@@ -561,7 +561,20 @@ async function statusHandler(message) {
           customCheckRunAnnotations = json;
         }
       } catch (e) {
-        await this.monitor.reportError(e);
+        if (e instanceof SyntaxError) {
+          let errorMessage = "returned text is not valid JSON";
```

Comment:
Let's add a bit more information here, including at least the artifact name and the taskId, so that whoever sees this comment has the tools they need to figure out what's wrong.

Reply:
Does this error message sound correct 
`Returned text is not valid JSON. Artifact Name - ${annotationsArtifactName}, TaskId - ${taskId}`

Reply:
I think "returned" is a little vague there.  How about "Custom annotations artifact ${annotationsArtifactName} on task ${taskId} does not contain valid JSON."